ee/server/service/vpp.go

```diff
@@ -232,6 +232,14 @@ func (svc *Service) GetAppStoreApps(ctx context.Context, teamID *uint) ([]*fleet
 		adamIDs = append(adamIDs, a.AdamID)
 	}
 
+	var iconsByAdamId map[string]fleet.SoftwareTitleIcon
```

Comment:
Given what this endpoint is doing (listing apps available from ABM), pretty sure we _don't_ want to override icons here; we should just return the Apple-supplied icon. This has the nice side effect of removing the need to have the `GetSoftwareTitleIconsByTeamAndAdamIDs` data store method (which is good, given my comments on that one).